lib/index.ts

```diff
@@ -148,39 +148,32 @@ export const imagePath = (app: string, file: string) => {
  * @return {string} URL with webroot for a file in an app
  */
 export const generateFilePath = (app: string, type: string, file: string) => {
-	const isCore = window?.OC?.coreApps?.indexOf(app) !== -1
+	const isCore = window?.OC?.coreApps?.includes(app)
+	const isPHP = file.slice(-3) === 'php'
 	let link = getRootUrl()
-	if (file.substring(file.length - 3) === 'php' && !isCore) {
-		link += '/index.php/apps/' + app
+	if (isPHP && !isCore) {
+		link += `/index.php/apps/${app}`
```

Comment:
template strings are more performant on modern browsers than string concatenation (and I think better readable).